app/src/apkinstaller.c

```diff
@@ -0,0 +1,157 @@
+#include "apkinstaller.h"
+
+#include "lockutil.h"
+#include "log.h"
+#include "command.h"
+#include <string.h>
```

Comment:
(nitpicking)

For consistency:
```
#include "apkinstaller.h"

#include <string.h>
#include "command.h"
#include "lockutil.h"
#include "log.h"
```
(first the associated .h, then the sorted `<>` includes, then the sorted `""` includes)